kubetest2/internal/deployers/eksapi/templates/unmanaged-nodegroup-efa.yaml

```diff
@@ -40,6 +40,10 @@ Parameters:
 
   UserData:
     Type: String
+  
+  CapacityReservationId:
+    Type: String
+    Description: Capacity reservation id for the unmanaged nodegroup
```

Comment:
Does CFN accept a blank string for this or do you get a 4xx?

Reply:
We will get 4xx, updated the PR to address it.